### ISSUE TITLE
add missing ENOMEDIUM to errno

### ISSUE THREAD

**Issue by radkujawa** [python/typeshed#3204]
reproduction:

file errno-enomedium.py:

```
import errno
print(errno.ENOMEDIUM)
```

```
$ python errno-enomedium.py 
123
$ mypy errno-enomedium.py
errno-enomedium.py:2: error: Module has no attribute "ENOMEDIUM"
$ python --version
Python 3.6.7
```

**Comment by srittau** [comment]
Patches welcome!

**Comment by Hanaasagi** [comment]
Following are available, but not included in typeshed and not documented in Python doc.
```
ENOMEDIUM
EMEDIUMTYPE
ENOKEY
EOWNERDEAD
EKEYREVOKED
ENOTRECOVERABLE
EKEYREJECTED
EKEYEXPIRED
ECANCELED
ERFKILL
```

**Comment by srittau** [comment]
Fixed by #3254
